New text on Colchicine (#832)
greenelab/covid19-review: content/20.pharmaceuticals.md

Its first potential mentions as a natural remedy for joint pain, rheumatoid pain and swelling are said to have appear in _Ebers Papyrus_ in 1500BC and the first application to treat gout is claimed to have been suggested in 550AD [@doi:10.1111/bjd.15896,@doi:10.2174/1871523011312010004, @doi:10.1002/art.1780130207]. 
P.S. Pelletier  and  J. Caventon isolated the drug in 1820 and P.L. Geiger purified its active ingredient and proposed the name colchicine in 1833 [@doi:10.2174/1871523011312010004].

Colchicine is representative of organic tropones and considered a high anti-inflamatory agent [@pmc:PMC1806919], but also is a well known and powerful  spindle  poison [@doi:10.1016/s1471-4892(01)00064] with multiple side effects reported [@pmc:PMC4825447] and high toxicity in high doses [@doi:10.2174/1871523011312010004].
Its mechanisms lie in affecing the protein threads connecting the centromere regions of chromosomes, what disrupts cell division. 
Successful treatment of COVID-19 with colchicine can be associated with quick decrease IL‐1β, IL‐18, and IL‐6 cytokines [@doi:0.1002/jcph.1645, @doi:10.1161/JAHA.115.002128 ]. 

> With interest of investigating potential COVID-19 anti-inflamatory action, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033]. 

Reviewer: ```suggestion
To investigate the potential of the anti-inflamatory properties on colchicine in COVID-19 patients, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033]. 
```

> In GRECCO-19, a randomized controlled trial with 105 participants (55 assigned to the colchicine group and 50 to the control group), colchicine lead to decreased  mortality  (14.0% for control group vs 1.8% in the colchicine group) and increased event-free survival (20.7 days vs 18.6) [@doi:10.1016/j.hjc.2020.03.002, doi:10.1001/jamanetworkopen.2020.13136, @clinicaltrial:NCT04326790]. 

Reviewer: ```suggestion
In GRECCO-19, a randomized controlled trial with 105 participants (55 assigned to the colchicine group and 50 to the control group), colchicine lead to decreased mortality (14.0% for the control group vs 1.8% in the colchicine group) and increased event-free survival (20.7 days vs 18.6 days) [@doi:10.1016/j.hjc.2020.03.002, doi:10.1001/jamanetworkopen.2020.13136, @clinicaltrial:NCT04326790]. 
```

> This evidence was further confirmed in much larget COLCORONA clinical trial [@clinicaltrial:NCT04322682].

Reviewer: ```suggestion
This evidence was confirmed in the much larger COLCORONA clinical trial [@clinicaltrial:NCT04322682].
```

> Among 4159 PCR confirmed COVID-19 cases, death or hospitalization due to COVID-19 infection in the 30 days following randomization appeared for 4.6% patients treated with colchicine vs 6.0% in placebo group.

Reviewer: ```suggestion
Among 4,159 PCR-confirmed COVID-19 cases, death or hospitalization due to COVID-19 infection in the 30 days following randomization occurred for 4.6% patients treated with colchicine vs 6.0% in the placebo group.
```

> The patients treated with colchicine experienced 25% lower hospitaliation rate, 50% less chances of necessity of mechanical ventilation and 44% lower mortality [@doi:10.1101/2021.01.26.21250494].

Reviewer: ```suggestion
The patients treated with colchicine experienced 25% lower hospitaliation rate, 50% lower chance of necessity of mechanical ventilation and 44% lower mortality [@doi:10.1101/2021.01.26.21250494].
```

> Two other trials are ECLA PHRI COLCOVID, which inspects co-administration of lopinavir/ritonavir [@clinicaltrial:NCT04328480], and ColCOVID-19 [@clinicaltrial:NCT04322565], have not provided their results yet. 

Reviewer: ```suggestion
Two other trials are ECLA PHRI COLCOVID, which inspects co-administration of lopinavir/ritonavir [@clinicaltrial:NCT04328480], and ColCOVID-19 [@clinicaltrial:NCT04322565] are ongoing and have not reported results yet. 
```
